README.md

```diff
@@ -42,31 +42,72 @@ jobs:
-
 ### Default behavior
 
-The default behavior of this action is the recommended workflow for a GitHub repository that contains Protobuf files.
+When you push a Git commit, tag, or branch to GitHub, the action will [push named modules to the BSR](https://buf.build/docs/bsr/module/publish) using `buf push`.
+
+On a pull request, the action will run all checks (using `buf build`, `buf lint`, `buf format`, `buf breaking`) and then post a [summary comment](#summary-comment) on the PR.
+
+When you delete a Git branch or tag, the action will archive the corresponding label on the BSR.
 
-| GitHub action event | Default behavior | `buf` commands |
-| - | - | - |
-| [`push`][push-event] | Modules that are configured with a BSR name are [pushed to the BSR](https://buf.build/docs/bsr/module/publish) every time a new Git commit, tag, or branch is pushed to GitHub. | `buf push` |
-| [`pull_request`][pull-request-event] | Run all checks and post (or update) a [summary comment](#summary-comment) on the PR every time the PR is updated. Errors are added as annotations on the PR. | `buf build`<br>`buf lint`<br>`buf format`<br>`buf breaking` |
-| [`delete`][delete-event] | Archive the corresponding label on the BSR every time a Git branch or tag is deleted from GitHub. | `buf beta registry archive --label` |
 
-### Skipping steps
+### Configuration
 
-The default configuration makes it possible to skip lint, formatting, or breaking change checks on a PR
-by adding a label with a (case-insensitive) special name to that PR.
+To customize the behavior of the action, you can set the following parameters in the workflow file.
+Add these parameters under the `with` section of the `uses` step in the workflow file.
+
+```yaml
+- uses: bufbuild/buf-action@v0.1
+  with:
+    ...
+```
 
-- `buf skip breaking`: skips breaking change detection.
-- `buf skip lint`: skips lint.
-- `buf skip format`: skips format. 
+| Parameter                       | Description                                        | Default            |
+|:--------------------------------|:---------------------------------------------------|:-------------------|
+| `version`                       | Version of the `buf` CLI to use. | Latest [version][buf-releases] |
+| `username`                      | Username for [logging into the BSR](https://buf.build/docs/bsr/authentication). | |
+| `token`                         | API token for [logging into the BSR](https://buf.build/docs/bsr/authentication). | |
+| `domain`                        | Domain for logging into the BSR, enterprise only.| `buf.build` |
+| `input`                         | [Input](https://buf.build/docs/reference/inputs) for the `buf` command. | |
+| `paths`                         | Limit to specific files or directories (separated by newlines). | |
+| `exclude_imports`               | Exclude imports. | |
+| `exclude_paths`                 | Exclude specific files or directories (separated by newlines). | |
```

Comment:
Exclude paths from what?

Going even further, perhaps neither of these options should exist either, as I think they only apply for lint and breaking (and build?)

Reply:
Exclude imports is a requested feature for buf-breaking-action: https://github.com/bufbuild/buf-breaking-action/issues/14